torch/csrc/jit/codegen/cuda/lower_allocation.cpp

```diff
@@ -590,7 +575,7 @@ class AllocationInserter : public kir::OptOutDispatch {
       // loops
       if (alloc.buffer->memoryType() == MemoryType::Shared &&
           !kir::ExpressionEvaluator::isConst(alloc.alloc_expr->size())) {
-        loop_nests_.insert(loop_nests_.begin(), alloc.alloc_expr);
+        exprs_.insert(exprs_.begin(), alloc.alloc_expr);
```

Comment:
Here, `exprs_` is modified, so using it with the for loop in `KirVisitor` seems invalid.